src/run.rs

```diff
@@ -0,0 +1,46 @@
+use chrono::prelude::*;
+use clap::ArgMatches;
+
+use crate::util::file::*;
+
+fn get_day(matches: &ArgMatches) -> u32
+{
+    if let Some(day) = matches.get_one::<String>("day")
+    {
+        day.parse().unwrap()
+    }
+    else
+    {
+        Utc::now().day()
+    }
+}
+
+pub fn run(matches: &ArgMatches)
+{
+    let day = get_day(matches);
+
+    let cwd = std::env::current_dir().unwrap();
+    let path = cargo_path(&cwd).unwrap_or(cwd);
+    let dir = day_path(path, day).unwrap();
+    let dir = dir.to_str().unwrap();
+
+    // check here that it actually exists
+    // ...
+
+    let res = std::process::Command::new("cargo")
+        .args(["run", "--bin", format!("day_{:02}", day).as_str()])
+        .current_dir(dir)
```

Comment:
Do we need to set current_dir🤔?

Reply:
Its so that the main file in the binary can access the input file as `input` and not `src/bin/day_01/input`.
Alternatively, you could pass the input file as an argument